penpost/penpost-web
# Gaj/feature/header

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 
Add a header component along with a hamburger menu component to display the dropdown of items a user can select.


Issues closed #

### Problems Encountered & Resulting Solutions  
> State any problems encountered 



> Describe your code changes in detail for reviewers.
Added a npm package for the hamburger menu, then added state w/hooks to conditionally render when the hamburger menu is toggled, it should show the dropdown items.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
Header component => look at the JS and scss files


### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start in the browser window


### Steps To Take
> Any questions and possible steps moving forward.
